src/applications/hca/components/VeteranInformation/AuthProfileInformation.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+import { normalizeFullName } from '../../utils/helpers';
+
+const AuthProfileInformation = ({ user }) => {
+  const { userFullName, dob } = user;
+  const veteranDOB = dob ? moment(dob).format('MMMM DD, YYYY') : null;
+  const veteranName = normalizeFullName(userFullName, true);
+  return (
+    <div className="vads-u-margin-top--2p5 vads-u-margin-bottom--2">
+      <p data-testid="hca-veteran-profile-intro">
+        {veteranDOB
+          ? 'This is the personal information we have on file for you.'
+          : 'Here’s the name we have on file for you.'}
+      </p>
+      <div className="vads-u-border-left--7px vads-u-border-color--primary vads-u-padding-y--1 vads-u-margin-bottom--3">
+        <dl className="vads-u-padding-left--1 vads-u-margin-y--0">
+          <div data-testid="hca-veteran-fullname">
+            <dt className="vads-u-visibility--screen-reader">Full name:</dt>
+            <dd className="vads-u-font-weight--bold">{veteranName}</dd>
```

Comment:
Does this need a `dd-privacy-hidden` class? I've also started adding a `data-dd-action-name` data attributes to replace the text in the event panel in Datadog.

Reply:
Adding privacy mask class to these! Great call!